README.md

```diff
@@ -29,7 +29,7 @@ You can add content by editing the `.html.md.erb` files. These files support con
 - HTML
 - Ruby
 
-👉 You can use Markdown and HTML to [generate different content types][example-content] and [Ruby partials to manage content][partials].
```

Comment:
Looks like [this file was deleted](https://github.com/alphagov/tdt-documentation/pull/127/files#diff-19535472ebe098550446abc0eeaab92893974cc2a1ce278d4c9db3563bdb2ac0L36) and moved to [a new file](https://github.com/alphagov/tdt-documentation/pull/127/files#diff-8cca47100766286315c6e134732526468608d6197a0015e40c12a5d215aeb008R19) . I'm not sure if it's still relevant for the frontend docs - if not, we could remove this bit.

Reply:
Yeah, I presume we're referring here to things that look like:

`<% partial foo/bar %>`

Which we don't use in the frontend docs.

I agree though - feels like if we're sending readers to the tech docs documentation anyway, we don't need this bit.

Reply:
Sounds good to me!